### PR TITLE
Migrate Showcase to PrimeFlex 3 (excluding PrimeFlex 2 - section)

### PR DESCRIPTION
**Pull request by christophs78**
Most of this was done via our CLI-tool.

### CODE REVIEW

**Review by melloware — APPROVED** [review]

**Comment by jepsar** [comment]
FYI Maybe there are more changes by other PRs, but https://github.com/primefaces/primefaces/pull/8214/files#diff-bbc5972b0bb140bfe69dc539ce9ea8879abb94e43807a7e4c3c28f975571e695 added PrimeFlex 2 classes to the showcase for sure.

**Comment by christophs78** [comment]
Yep. Not good to keep PRs open for such a long time.
I won´t keep it alive for months.

**Comment by christophs78** [comment]
Now it´s rebased and merge-able again.
To catch all the PrimeFlex 2 - based additions/changes done to Showcase since december 2021 i can offer to do another PR which tries to catch this after this PR is merged.

File: primefaces-showcase/src/main/webapp/ui/input/oneRadio.xhtml [hunk 1]
```diff
@@ -34,8 +34,8 @@
     <ui:define name="implementation">
         <div class="card">
             <h:form>
-                <h5 class="mt-0">Basic</h5>
-                <p:selectOneRadio id="console" value="#{radioView.console}" unselectable="true">
+                <h5 class="p-mt-0">Basic</h5>
```
Comment (jepsar) [inline primefaces-showcase/src/main/webapp/ui/input/oneRadio.xhtml]: This seems to be wrong (probably due to a conflict)

Comment (melloware) [reply primefaces-showcase/src/main/webapp/ui/input/oneRadio.xhtml]: If you want you can edit this PR and fix it. Yeah it was a conflict I was trying to resolve

Comment (jepsar) [reply primefaces-showcase/src/main/webapp/ui/input/oneRadio.xhtml]: Will have a look on my laptop

**Comment by mertsincan** [comment]
Wooowww! Great work ;) If you update primeflex.css directly instead of org.webjars.npm everything is ok for me. Thanks a lot for your contribution! 
Also, we can remove PrimeFlex 2 section. Please check PrimeFlex section in https://www.primefaces.org/primereact/setup/

**Comment by melloware** [comment]
@christophs78 i will take care of this for you if you don't mind!

**Pull request merged**